src/CoreFx.Private.TestUtilities/src/System/PlatformDetection.Windows.cs

```diff
@@ -268,39 +224,8 @@ private static unsafe int GetWindowsBuildNumber()
             out int pdwReturnedProductType
         );
 
-        [DllImport("ntdll.dll", ExactSpelling=true)]
-        private static extern int RtlGetVersion(ref RTL_OSVERSIONINFOEX lpVersionInformation);
```

Comment:
We can eliminate the duplication just by having this project include the interop file.  We don't need to move the GetWindows* helper implementations into that interop file; they can remain here in this test project, so as not to clutter up that shared file, which is also included in production libs.